Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -3460,7 +3468,11 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
 
       if let calledMemberAccessExpr = calledExpression.as(MemberAccessExprSyntax.self) {
         if calledMemberAccessExpr.base != nil {
-          before(calledMemberAccessExpr.dot, tokens: [.break(.contextual, size: 0)])
+          if Syntax(calledMemberAccessExpr).isPostfix {
```

Comment:
Thinking about it more, this logic is probably incorrect and won't indent postfix member access even when there isn't a `#if`. I'll add a scenario to test that if one doesn't already exist.

Without this conditional I saw the following output:

```swift
...
  Text("")
  #if os(iOS)
    .iOSSpecificModifier()  
      .anotherModifier() // this line has extra indentation
  #endif
  .commonModifier()
...